service/controller/todo.go

```diff
@@ -38,7 +38,7 @@ func NewTODO(config TODOConfig) (*TODO, error) {
 			K8sClient: config.K8sClient,
 			Logger:    config.Logger,
 			NewRuntimeObjectFunc: func() runtime.Object {
-				return new(corev1.Pod)
+				return new(v1alpha1.App)
```

Comment:
Here's the only change outside dependencies. But I also wanted apiextensions pulled.